tools/scripts/verify_prism_regression_tests.sh

```diff
@@ -17,29 +18,59 @@ for file in test/prism_regression/*.rb; do
   ./bazel-bin/main/sorbet --stop-after=parser --print=parse-tree "$file" > temp_parse_tree.txt 2> /dev/null
   set -e # Re-enable exit on error
 
-  # Compare with existing parse tree
-  if diff -q "temp_parse_tree.txt" "test/prism_regression/${file_name}.parse-tree.exp" > /dev/null; then
+  # Generate desugar tree
+  set +e # Disable exit on error for the next command
+  ./bazel-bin/main/sorbet --stop-after=desugarer --print=desugar-tree-raw "$file" > temp_desugar_tree.txt 2> /dev/null
+  set -e # Re-enable exit on error
```

Comment:
You might consider:

* Combining these `set`s with the one above; and/or
* Just invoking `sorbet` once with `sorbet --stop-after=desugarer --print=parse_tree:temp_parse_tree.txt --print=desugar-tree-raw:temp_desugar_tree.txt 2>/dev/null` or somesuch

The latter can be helpful for making tests run faster in debug builds.